docs/technical_reference/flowsheets/mvc.rst

```diff
@@ -0,0 +1,126 @@
+Mechanical Vapor Compression
+============================
+
+Introduction
+------------
+
+Mechanical Vapor Compression (MVC) desalination is an evaporative, compressor-driven process where vapor generated
+from saline feedwater is compressed to raise its temperature and pressure. This compressed vapor then condenses,
+releasing latent heat to evaporate the feedwater, thus producing distillate without external heating or cooling.
+The system primarily consists of an evaporator/condenser and a compressor, with optional heat exchangers for enhanced
+heat integration, and may utilize external steam to initiate the process.
+
+Implementation
+--------------
+
+Figure 1 illustrates the process flow diagram for a Mechanical Vapor Compression (MVC) desalination system.
+In this configuration, feedwater is pumped and equally divided between the cold inlets of a distillate heat
```

Comment:
The feed is not necessarily equally divided between the two heat exchangers because the split fraction (i.e. proportion of feed directed to the distillate heat exchanger) is set equal to the water recovery in a constraint.